corehq/apps/cloudcare/templates/formplayer/dependencies.html

```diff
@@ -3,6 +3,11 @@
 
   <script src='https://api.mapbox.com/mapbox.js/v3.3.1/mapbox.js' async></script>
   <link href='https://api.mapbox.com/mapbox.js/v3.3.1/mapbox.css' rel='stylesheet' />
+  <script
+    src="https://browser.sentry-cdn.com/7.16.0/bundle.min.js"
```

Comment:
Is it possible to pull this from yarn? [This page](https://docs.sentry.io/platforms/javascript/) has a yarn option (though I didn't look closely enough to confirm it's the same library).

Reply:
(nevermind, missed your comment on this)